SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -434,6 +452,9 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_AUTH_SERVER_INTEREST =
 								// Actor tags
 								ACTOR_TAG_COMPONENT_ID, ACTOR_AUTH_TAG_COMPONENT_ID,
 
+								// Strategy Worker LB components
+								ACTOR_SET_MEMBER_COMPONENT_ID, ACTOR_GROUP_MEMBER_COMPONENT_ID,
```

Comment:
We can remove this

Reply:
This comment still valid?

Reply:
It's a part of "If we could dynamically modify interest, these are only here for the test." which hasn't changed, sadly :/